[Monitors] Document cumulative evaluation windows (#15383)
DataDog/documentation: content/en/monitors/create/configuration.md

### Evaluation window

> Monitor can be evaluated at a certain frequency in two different ways:

Reviewer: ```suggestion
A monitor can be evaluated using rolling time windows, or cumulative time windows.
```

> **Rolling time windows**
> A rolling time window has a fixed size and moves its starting point over time. Monitors support looking back at the last `5 minutes`, `15 minutes`, `1 hour`, and more custom options.

Reviewer: ```suggestion
#### Rolling time windows

A rolling time window has a fixed size and moves its starting point over time. Monitors support looking back at the last `5 minutes`, `15 minutes`, `1 hour`, or over a custom specified time window.
```

> **Cumulative time windows**

Reviewer: ```suggestion
#### Cumulative time windows
```

A cumulative time window has a fixed starting point and expands over time. Monitors support three different cumulative time windows:

> - `Current hour`: Time window of maximum one hour starting at a configurable minute of an hour. For example, monitor amount of calls an HTTP endpoint receives in one hour starting at minute 0.

Reviewer: ```suggestion
- `Current hour`: A time window with a maximum of one hour starting at a configurable minute of an hour. For example, monitor amount of calls an HTTP endpoint receives in one hour starting at minute 0.
```

> - `Current day`: Time window of maximum 24 hours starting at a configurable hour and minute of a day. For instance, monitor a [daily log index quota](https://docs.datadoghq.com/logs/log_configuration/indexes/#set-daily-quota) by using the `current day` time window and letting it start at 2:00pm UTC.

Reviewer: ```suggestion
- `Current day`: A time window with a maximum of 24 hours starting at a configurable hour and minute of a day. For example, monitor a [daily log index quota](https://docs.datadoghq.com/logs/log_configuration/indexes/#set-daily-quota) by using the `current day` time window and letting it start at 2:00pm UTC.
```

- `Current month`: Looks back at the current month starting on the first of the month at midnight UTC. This option represents a month-to-date time window.

{{< img src="/monitors/create/cumulative_evaluation_window.png" alt="Cumulative Evaluation Window" style="width:100%;">}}

> A cumulative time window is _reset_ once its maximum time span is reached. For instance, a cumulative time window looking at the `current month` resets itself on the first of each month at midnight UTC. Alternatively, a cumulative time window of `current hour` which starts at minute 30 resets itself every hour, for example at 6:30am, 7:30am, 8:30am, etc.

Reviewer: ```suggestion
A cumulative time window is reset once its maximum time span is reached. For example, a cumulative time window looking at the `current month` resets itself on the first of each month at midnight UTC. Alternatively, a cumulative time window of `current hour`, which starts at minute 30, resets itself every hour. For example, at 6:30am, 7:30am, 8:30am, etc.
```
